helloworld-simple-project/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,16 @@
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
```

Comment:
Last time io remember suggesting sth about passing a parameter here and make it say `Hello Shatakshi` or sth for example. Can you try this out? Try to do this test-driven (via TDD I mean), so you should start with a test. I see that there are already some test skeleton is created , you can use that test classes and start with adding some more functions.

Reply:
Yes I will writing a different application for that and will commit it here.